### PR TITLE
Sa'idu Hassan

### PR DESCRIPTION
**Pull request by SaiduHassan**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by mexsoldier** [comment]
> * [x]  Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
> * [x]  Have you explained what your changes do, and why they add value to the Guides?
> 
> **Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

**Comment by SaiduHassan** [comment]
I want contribute

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
```
Comment (SaiduHassan) [inline index.html:1]: Hello how are you

Comment (SaiduHassan) [inline index.html:1]: Hey

**Review by SaiduHassan — COMMENTED** [review]
Thanks

File: index.html (hunk 1 above)
Comment (MaryJaneLove4) [reply index.html:1]: AM FAIN. and you?

Comment (MaryJaneLove4) [reply index.html:1]: What?

**Comment by marnad7** [comment]
Huh?

**Comment by lordany** [comment]
What yo folks going on 'bout

**Pull request closed without merging**